Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG BASEIMAGE=rust:latest
+
+# BUILDER PATTERN
+
+FROM $BASEIMAGE AS builder
+
+WORKDIR /builder
+
+COPY . .
```

Comment:
Check if all files are needed. Else create a `.dockerignore`.

Reply:
DId not know such option existed, i'll dig in.